tools/scripts/platform/xilinx/util.tcl

```diff
@@ -242,7 +242,11 @@ proc upload {} {
 	set cpu [_get_processor]
 
 	# Connect to the fpga
-	connect
+  if { [info exists ::env(XSCT_REMOTE_HOST)] || [info exists ::env(XSCT_REMOTE_PORT)] } {
```

Comment:
would && make more sense here ? they both need to be provided for it to work i assume

Reply:
Yes, you are correct, I will fix this